hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenCreateValidator.java

```diff
@@ -192,4 +198,13 @@ public void validateAssociation(
         validateTrue(
                 tokenRelStore.get(account.accountId(), token.tokenId()) == null, TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT);
     }
+
+    /**
+     * Validate the metadata size for token create operations.
+     * @param metaData The metadata of the tokens to create.
+     * @param maxMetadataBytes The max metadata size for token create based on config.
```

Comment:
```suggestion
    /**
     * Validate the size of metadata being set for the token to be created.
     * @param metaData The metadata of the created token.
     * @param maxMetadataBytes The maximum number of bytes allowed for metadata to be set on the token
```

Reply:
That verbiage makes a lot more sense. Thanks @Neeharika-Sompalli !